index.html

```diff
@@ -53,9 +53,9 @@
             const urlElevationLayers = [];
 
             urlImageryLayers.push('examples/layers/JSONLayers/Ortho.json');
-            urlImageryLayers.push('examples/layers/JSONLayers/OrthosCRS.json');
-            urlImageryLayers.push('examples/layers/JSONLayers/ScanEX.json');
-            urlImageryLayers.push('examples/layers/JSONLayers/Region.json');
+            // urlImageryLayers.push('examples/layers/JSONLayers/OrthosCRS.json');
```

Comment:
The external returns on the demo are:
  * Loading speed (due to the 4 layers)
  * Unnecessary layers: region (ugly), CRS (lined with ortho) and scan (limited to France)